src/instanceOfComponent.js

```diff
@@ -27,6 +27,28 @@ const instanceOfComponentFactory = (Component, isRequired) => (
     return null
 }
 
+/**
+ * Ensure the prop value is an instance of a certain component
+ * @param {function} Component - The component that is expected
```

Comment:
`function` could be `React.Component`.
React's `SFC`s will be wrapped in a Component class internally as well.

Reply:
Yeah I guess that is more descriptive... But technically it just works like this...
```javascript
class Test {}
console.log(typeof Test)
// 'function'
const Test2 = () => {}
console.log(typeof Test2)
// 'function'
```